rest/route/host_spawn.go

```diff
@@ -267,7 +267,7 @@ func (h *hostStopHandler) Parse(ctx context.Context, r *http.Request) error {
 		SubscriptionType string `json:"subscription_type"`
 	}{}
 	if err := util.ReadJSONInto(body, &options); err != nil {
-		return errors.Wrap(err, "Argument read error")
+		h.subscriptionType = ""
```

Comment:
Would this actually error if the option is not present. (Is this because there would be no document at all?)

Reply:
in the current CLI the request body contains a string (the empty string), and attempting to read into the struct errors.

I tried it out in staging:
```
> evergreen -c ~/.evergreen.yml_staging host stop --host i-04989d25711a819ad
A new version is available. Run 'evergreen get-update --install' to download and install it.
problem stopping host: 400 (Bad Request): json: cannot unmarshal string into Go value of type struct { SubscriptionType string "json:\"subscription_type\"" }
```